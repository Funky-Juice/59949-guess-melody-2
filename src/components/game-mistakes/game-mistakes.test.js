import renderer from 'react-test-renderer';
import GameMistakes from './game-mistakes';

it(`GameMistakes correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<GameMistakes errors={0}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
