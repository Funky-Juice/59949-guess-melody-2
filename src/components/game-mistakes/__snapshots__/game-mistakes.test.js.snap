// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameMistakes correctly renders after relaunch 1`] = `
<div
  className="game__mistakes"
/>
`;
